### PR TITLE
added gdpr consent compliance

### PR DESCRIPTION
**Pull request by brittanyzellman**
Added the gdpr relevant code. Adds the applies boolean and consent string to the params obj from the getConsent method provided by Index.

### CODE REVIEW

File: triple-lift-htb.js [hunk 1]
```diff
@@ -16,14 +16,14 @@
 // Dependencies ////////////////////////////////////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////
 
-var Browser = require('browser.js');
```
Comment (ix-certification) [inline triple-lift-htb.js]: Please add this back. Any reason why this was removed?

File: triple-lift-htb.js [hunk 2]
```diff
@@ -80,6 +81,7 @@ function TripleLiftHtb(configs) {
      * @private {object}
      */
     var __baseUrl;
+    var gdprStatus = ComplianceService.gdpr.getConsent();
```
Comment (ix-certification) [inline triple-lift-htb.js]: Please move to this to inside `__generateRequestObj`

**Comment by ix-certification** [comment]
Please add `ComplianceService.isPrivacyEnabled()` as this is the expected behaviour of the wrapper.

**Comment by brittanyzellman** [comment]
The above method returns a boolean indicating whether the publisher is integrated with a CMP and asked for consent? I don't see anything here https://github.com/indexexchange/ix-base-adapter/commit/58197f618a6d3f6ec86d625da70cc2f946e9a88a about where this should live. Should it be something like:

`if (ComplianceService.isPrivacyEnabled()) {
    var gdprStatus = ComplianceService.gdpr.getConsent();
}`

**Comment by ix-certification** [comment]
Hi @brittanyzellman,

We reviewed the latest changes and was able to successfully build a test wrapper. We were able to confirm the `gdpr applies` & the `consentString` values being passed into the ad request call.

Please refer to the `test-wrappers` branch for steps on how to test on your end. If everything looks good on your end we are ready to merge. Let us know if you have any questions.

**Pull request merged**